### DISCUSSION TITLE
Say Hi to Ensembles first alpha

### DISCUSSION THREAD

**Discussion by SubhadeepJasu** [ensemblesaw/ensembles-app#11] [Announcements]
**This is the first public alpha release of Ensembles.**

Known Issues and incomplete features:
- Style Engine only supports 4/4 time signature styles
- Style Engine player loops a little later than usual after the specific time stamp
- No way to manually change tempo without changing styles
- Style player volume cannot be changed
- Scroll wheel beside the display doesn't work and is still very choppy.
- Menu up, down and activate buttons don't work.
- Arpeggiator and Harmonizer doesn't work
- Reverb and  Chorus buttons don't turn off reverb and chorus respectively
- Multipad variables not implemented
- Registration Memory not implemented yet
- Hold toggle doesn't work
- Keyboard styling isn't final
- No app settings or preferences UI yet

<hr /><em>This discussion was created from the release <a href='https://github.com/SubhadeepJasu/Ensembles/releases/tag/v0.0.1'>Say Hi to Ensembles first alpha</a>.</em>

**Comment by ghost** [comment]
I'm impressed! Wishing you guys well! P.S. you should monetize in Elementary OS AppCenter!

**Reply by SubhadeepJasu** [reply]
Thanks! AppCenter monetization isn't working just yet for all the new apps. I am waiting for them to get the system to working and then the app will be monetized from the beta version or final release depending on when the system gets fixed.
